src/main/java/com/google/sps/servlets/UserComment.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.api.client.util.DateTime;
+import com.google.api.services.youtube.model.Comment;
+import com.google.api.services.youtube.model.CommentThread;
+
+public class UserComment {
+  private String commentId;
+  private String commentMsg;
+  private DateTime publishDate;
```

Comment:
Add a double property for sentiment score

Reply:
Also agreed